app/components/project-post-list.js

```diff
@@ -1,25 +1,88 @@
 import Ember from 'ember';
 
+/**
+  `project-post-list` provides a list of posts within the project and allows filtering and paging.
+
+  ## Default usage
+  ```Handlebars
```

Comment:
also I think this has to be lower case for it to work (haven't checked, so it might be a good idea to do so) it sets the syntax highlighting for this.